renderer/src/components/Dashboard.jsx

```
@@ -526,28 +526,28 @@ export default function Dashboard() {
               <div className="bg-white p-4 rounded-md shadow-sm border border-gray-100 flex items-center justify-between">
-                <span className="font-medium text-gray-700">Killmail Charts</span>
+                <span className="font-medium text-gray-700">Kill Charts</span>
                 <div className="flex items-center space-x-2">
                   <span className="bg-green-100 text-green-800 text-sm px-2 py-1 rounded">
                     Enabled
                   </span>
                   <button
                     className="relative group p-2 text-gray-600 hover:bg-gray-200 rounded-md transition-colors"
-                    onClick={() => window.open('/charts', '_blank')}
+                    onClick={() => window.open('/kill-charts', '_blank')}
                   >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)